cif2cell/cif2cell.xml

```diff
@@ -0,0 +1,46 @@
+<tool id="cif2cell" name="Convert .cif file to .cell (cif2cell)" version="0.1.0" python_template_version="3.5">
+    <command detect_errors="exit_code"><![CDATA[
+        pip install cif2cell==2.0.0a3 &&
+        ln -s $cif_structure $cif_structure.name &&
+        cif2cell -f $cif_structure.name -p castep -o out.cell &&
+        sed -i '1,8d' out.cell
```

Comment:
I don't think deleting these comments at the start of the file is a good idea – they give credit to cif2cell where it's needed. Always best to keep credit with/in the output file where possible, as these files may be taken and used elsewhere.

Reply:
The issue here is cif2cell produces a timestamp as part of the comments, so I had to delete them to make the tests pass